### PR TITLE
Add support chart-chart filtering in wizard

### PR DESCRIPTION
**Pull request by kuzmadom**

### CODE REVIEW

**Review by korvin89 — APPROVED** [review]

File: src/ui/libs/DatalensChartkit/ChartKit/helpers/apply-hc-handlers.ts [hunk 1]
```diff
@@ -234,6 +30,19 @@ export const fixPieTotals = (args: {data: GraphWidget}) => {
     });
 };
 
+function getActionParams(params: Record<string, string | string[]> = {}) {
```
Comment (Marginy605) [inline src/ui/libs/DatalensChartkit/ChartKit/helpers/apply-hc-handlers.ts:33]: We have already functions in dashkit for getting actionParams from params, see `pickActionParamsFromParams`. And it has different formats to return (with prefix or not).

File: src/ui/components/Widgets/Chart/Chart.tsx [hunk 2]
```diff
@@ -170,7 +170,7 @@ export const Chart = (props: ChartNoWidgetProps) => {
         ignoreUsedParams,
         clearedOuterParams,
         onInnerParamsChanged,
-        enableActionParams: true,
+        enableActionParams: false,
```
Comment (Marginy605) [inline src/ui/components/Widgets/Chart/Chart.tsx:173]: For this change we need to add new reserved get-parameter to enable it in preview. And add it;
- for forming link to open in new tab (and remove from link for edit, as far as I remember, there is single helper for both)
- support force adding get-parameter in mailings

**Review by Marginy605 — CHANGES_REQUESTED** [review]

File: src/ui/components/Widgets/Chart/Chart.tsx (hunk 2 above)
Comment (kuzmadom) [reply src/ui/components/Widgets/Chart/Chart.tsx]: Added a separate parameter `_ap_enabled` (used for preview). The link generation for screenshots will be done in a separate task.

File: src/ui/constants/common.ts [hunk 3]
```diff
@@ -255,6 +255,7 @@ export const URL_OPTIONS = {
     EMBEDDED: '_embedded',
     NO_CONTROLS: '_no_controls',
     LANGUAGE: '_lang',
+    ACTION_PARAMS_ENABLED: '_ap_enabled',
```
Comment (Marginy605) [inline src/ui/constants/common.ts:258]: Bad name - dashkit will think that there is action param `enabled` for this chart

**Review by Marginy605 — APPROVED** [review]

**Pull request merged**